build.ps1

```diff
@@ -83,7 +85,7 @@ function MD5HashFile([string] $filePath)
     }
     finally
     {
-        if ($null -ne $file)
```

Comment:
Please revert, will make updating to newer versions of this script easier.

Reply:
I dont think I changed this file